tests/integration/milvus_memory_tests.py

```
@@ -51,7 +51,7 @@ def test_get_relevant(self) -> None:
             self.assertEqual(len(relevant_texts), k)
             self.assertIn(self.example_texts[1], relevant_texts)
 
-except:
+except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)